scripts/helpers.php

```
@@ -0,0 +1,292 @@
+    foreach ($arguments as $argument) {
+        if (in_array($argument, ['-ci', '--dry-run'], true)) {
+            $dryRun = true;
+
+            continue;
+        }
+
+        if (str_starts_with($argument, '-')) {
+            $options[] = $argument;
+        } else {
+            $paths[] = $argument;
+        }
```

PHPMD (2.15.0)

[error] The method split_script_arguments uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)